content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -9,197 +9,82 @@ card:
 
 {{% capture overview %}}
 
-This page shows you how to install [Minikube](/docs/tutorials/hello-minikube), a tool that runs a single-node Kubernetes cluster in a virtual machine on your personal computer.
+This page shows you how to install [minikube](https://minikube.sigs.k8s.io/), a tool that runs a Kubernetes cluster on your personal computer.
 
 {{% /capture %}}
 
 {{% capture prerequisites %}}
 
-{{< tabs name="minikube_before_you_begin" >}}
-{{% tab name="Linux" %}}
-To check if virtualization is supported on Linux, run the following command and verify that the output is non-empty:
-```
-grep -E --color 'vmx|svm' /proc/cpuinfo
-```
-{{% /tab %}}
+To run minikube, you'll need:
 
-{{% tab name="macOS" %}}
-To check if virtualization is supported on macOS, run the following command on your terminal.
-```
-sysctl -a | grep -E --color 'machdep.cpu.features|VMX'
-```
-If you see `VMX` in the output (should be colored), the VT-x feature is enabled in your machine.
-{{% /tab %}}
-
-{{% tab name="Windows" %}}
-To check if virtualization is supported on Windows 8 and above, run the following command on your Windows terminal or command prompt.
-```
-systeminfo
-```
-If you see the following output, virtualization is supported on Windows.
-```
-Hyper-V Requirements:     VM Monitor Mode Extensions: Yes
-                          Virtualization Enabled In Firmware: Yes
-                          Second Level Address Translation: Yes
-                          Data Execution Prevention Available: Yes
-```
-
-If you see the following output, your system already has a Hypervisor installed and you can skip the next step.
-```
-Hyper-V Requirements:     A hypervisor has been detected. Features required for Hyper-V will not be displayed.
-```
-
-
-{{% /tab %}}
-{{< /tabs >}}
+* A host with administrator privileges (for example, ability to `sudo` or `Run as Administrator`)
```

Comment:
How about making this be after the “compatible container or virtual machine manager” point, as a nested item, and explain that if your VMM / container runtime needs you to have admin access then you will need it (for example: VirtualBox and KVM usually don't need you to have superuser access).
If someone does not have this access, I'm concerned that they might see this and incorrectly infer that Minikube does not meet their needs, even if it would in fact work.

I'm not sure if Minikube automatically selects a suitable runtime based on its guess as to whether you have that access. That might be a note to mention if it does not.

Reply:
As currently written, the installation instructions all require administrative access to install, since they either call `sudo` or interact with a package manager which requires administrative access.

There are edge cases where a user could feasibly install minikube without administrative access, but it would involve straying from these instructions.